casepro/statistics/signals.py

```diff
@@ -52,3 +78,35 @@ def record_incoming_labelling(sender, instance, action, reverse, model, pk_set,
     elif action == 'pre_clear':
         for label in instance.labels.all():
             DailyCount.record_removal(day, DailyCount.TYPE_INCOMING, label)
+
+
+@receiver(post_save, sender=CaseAction)
+def record_new_case_action(sender, instance, created, **kwargs):
+    org = instance.case.org
+    user = instance.created_by
+    partner = instance.case.assignee
+    case = instance.case
+
+    day = datetime_to_date(instance.created_on, instance.case.org)
+
+    if instance.action == CaseAction.CLOSE:
+        # count the time to close on an org level
+        td = instance.created_on - case.opened_on
+        minutes_since_open = ceil(td.total_seconds() / 60)
```

Comment:
Still don't feel great about use of minutes for timing things - this adds an average of 30 seconds.